src/Storages/StorageReplicatedMergeTree.cpp

```diff
@@ -834,6 +836,9 @@ bool StorageReplicatedMergeTree::createTableIfNotExists(const StorageMetadataPtr
         ops.emplace_back(zkutil::makeCreateRequest(replica_path + "/mutation_pointer", "",
             zkutil::CreateMode::Persistent));
 
+        ops.emplace_back(zkutil::makeCreateRequest(replica_path + "/creator_info", toString(getStorageID().getFullTableName()) + "|" + toString(ServerUUID::get()),
```

Comment:
We must use table `uuid` as creator_info, not name. It will break after first rename query.

Reply:
The main goal for this fix is to fix the replication queue for the Replicated database, which becomes blocked after the CREATE TABLE query has partially failed. In this case even after we rename the table on other nodes we will still first try to create it with the same name and the same UUID, so it won't break after the first rename.

However I checked once again and in case we partially created the table under some name, then the replica went offline and returned when the log was already far away and we would need to `recoverLostReplica()`, considering only the table name changed and not the column metadata, we could solve this case at least.

So will change to UUID.

Reply:
The main goal for this fix is to fix the replication queue for the Replicated database, which becomes blocked after the CREATE TABLE query has partially failed. In this case even after we rename the table on other nodes we will still first try to create it with the same name and the same UUID, so it won't break after the first rename.

However I checked once again and in case we partially created the table under some name, then the replica went offline and returned when the log was already far away and we would need to `recoverLostReplica()`, considering only the table name changed and not the column metadata, we could solve this case at least.

The table name in `creator_info` on the other hand resovles a case in the test where we create the table with a static path without stating UUID which (as I understand) will be random with each query. 

So I am not sure, but probably will change it to table UUID.

Reply:
Yes, let's change :)